package.json

```diff
@@ -3,14 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "concurrently": "^4.1.0",
```

Comment:
Nice touch. I like this a lot. I can't tell you how many times a student has come to me for help only to find out they didn't start json-server.